plugins/catalog-backend/src/permissions/rules/isEntityOwner.ts

```diff
@@ -28,7 +29,10 @@ export const isEntityOwner = createCatalogPermissionRule({
   name: 'IS_ENTITY_OWNER',
   description: 'Allow entities owned by the current user',
   resourceType: RESOURCE_TYPE_CATALOG_ENTITY,
-  apply: (resource: Entity, claims: string[]) => {
+  schema: z.object({
+    claims: z.array(z.string()),
```

Comment:
Again, just noting the description todo. I wonder if we can encode the expectation that each property has a description in the type somehow?